lib/geo.ex

```diff
@@ -0,0 +1,155 @@
+defmodule Geo do
+  @type continent :: :europe | :northamerica | :africa | :australia | :asia | :other
+
+  # Draw using https://www.calcmaps.com/map-radius/
+
+  # Europe
+  # Radius: 2181215 m | 2181.21 km | 1355.34 mi | 7156217 ft | 1177.76 nm
+  # Circle Area: 14946750746399 m2 | 14946750.75 km2
+  # Lat,Lon: 56.54461,10.91914
+
+  # Spain and Portugal
+  # Radius: 591047 m | 591.05 km | 367.26 mi | 1939130 ft | 319.14 nm
+  # Circle Area: 1097472125815 m2 | 1097472.13 km2
+  # Lat,Lon: 41.17816,-4.24584
+
+  defp isInEurope(latitude, longitude) do
+    europe = %Geocalc.Shape.Circle{
+      latitude: 56.54461,
+      longitude: 10.91914,
+      radius: 2_181_215
+    }
+
+    spainAndPortugal = %Geocalc.Shape.Circle{
+      latitude: 41.17816,
+      longitude: -4.24584,
+      radius: 591_047
+    }
```

Comment:
Was necessary to use a circle shape since to include Spain and Portugal in Europe